lms/envs/common.py

```diff
@@ -960,6 +960,16 @@
     # .. toggle_target_removal_date: None
     # .. toggle_tickets: 'https://openedx.atlassian.net/browse/MST-1348'
     'ENABLE_INTEGRITY_SIGNATURE': False,
+
+    # .. setting_name: FEATURES['ENABLE_NEW_BULK_EMAIL_EXPERIENCE']
+    # .. setting_default: False
+    # .. setting_description: When true, replaces the bulk email tool found on the
+    #   instructor dashboard with a link to the new communications MFE version instead.
+    #   Stting the tool to false will leave the old bulk email tool experience in place.
+    # .. toggle_creation_date: 2022-03-18
+    # .. toggle_target_removal_date: None
+    # .. toggle_tickets: 'https://openedx.atlassian.net/browse/MICROBA-1758'
```

Comment:
Nit: I believe We are missing part of the docstring, `toggle_implementation`, which would be a `DjangoSetting`. I think a few of the settings are named incorrectly (`setting_name` vs. `toggle_name`. It should match the above examples. See https://edx.readthedocs.io/projects/edx-toggles/en/latest/how_to/documenting_new_feature_toggles.html for an example doc template of a new setting.

IIRC, the names are important as they are consumed by the `edx-toggles` repo and it generates a report of all the edx-platform toggles and their values. See https://github.com/openedx/edx-toggles for more.